conduit/errors.go

```diff
@@ -0,0 +1,11 @@
+package conduit
+
+import "fmt"
+
+// CriticalError an error that causes the entire conduit pipeline to
+// stop
+type CriticalError struct{}
```

Comment:
This looks unused as of now.

Reply:
Yeah, this is just here for future development.